README.md

```diff
@@ -79,6 +102,8 @@ Within a segment, the following can be used:
 
 **Nonterminals** are written as `|x|`, `|x_opt|`, `|x[p]|`, or `|x[p]_opt|`. These are translated, respectively, into `<emu-nt>x</emu-nt>`, `<emu-nt optional>x</emu-nt>`, `<emu-nt params="p">x</emu-nt>`, or `<emu-nt params="p" optional>x</emu-nt>`. Nonterminal names can only be composed of letters. Params can be composed of anything except a closing square bracket.
 
+You can escape any format above with a backslash. In order to include a literal backslash, you must escape it (eg. \\). Escaping of any non-format characters will not be considered an escape and will render literally (eg. `\a` simply renders as `\a`).
+
 Finally, anything in between `<` and `>` will count as an **html tag** and will not be messed with, so e.g. `<span title="_x_">` will be left alone instead of being translated into `<span title=<code>"<var>x</var>"</code>>`.
 
 ## Interaction with Ecmarkup
```

Comment:
"the introduction of the Ecmarkdown document production" (below this) is a bit outdated. Maybe replace it with something more nebulous or delete that clause entirely.